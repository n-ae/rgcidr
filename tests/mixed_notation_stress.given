192.168.1.1
::192.168.1.1
2001:db8::192.168.1.1
::ffff:192.168.1.1
64:ff9b::192.168.1.1
2001:0db8:85a3:0000:0000:8a2e:0370:192.168.1.1
fe80::192.168.1.1
192:168:1:1
192:168:1:1::
::192:168:1:1
2001:192:168:1
2001:0192:0168:0001::
::2001:192:168:1
192.168.1.1:2001:db8
192.168.1.1::2001
2001::192.168.1.1
::2001:db8:192.168.1.1
192.168.1.1/32
2001:db8::1/128
::ffff:192.168.1.1/128
192.168.1.1-192.168.1.10
2001:db8::1-2001:db8::10
[192.168.1.1]
[2001:db8::1]
192.168.001.001
2001:0db8:0000:0000:0000:0000:0192:0168
text192.168.1.1
2001:db8::1text
text
192.168.1.300
2001:db8::gggg
127.0.0.1:8080
[2001:db8::1]:8080
2001:db8::1%eth0
