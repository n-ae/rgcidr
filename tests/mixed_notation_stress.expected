2001:db8::192.168.1.1
2001:db8::1/128
2001:db8::1-2001:db8::10
[2001:db8::1]
2001:0db8:0000:0000:0000:0000:0192:0168
[2001:db8::1]:8080
2001:db8::1%eth0
